src/library_noderawfs.js

```diff
@@ -10,35 +10,27 @@ addToLibrary({
     if (!ENVIRONMENT_IS_NODE) {
       throw new Error("NODERAWFS is currently only supported on Node.js environment.")
     }
-    // Use this to reference our in-memory filesystem
-    var VFS = Object.assign({}, FS);
-    // Override the init function with our own
-    FS.init = NODERAWFS.init;`,
-  $NODERAWFS: {
-    init() {
-      var _wrapNodeError = function(func) {
-        return function(...args) {
-          try {
-            return func(...args)
-          } catch (e) {
-            if (e.code) {
-              throw new FS.ErrnoError(ERRNO_CODES[e.code]);
-            }
-            throw e;
+    var _wrapNodeError = function(func) {
```

Comment:
It strange, the github UI won't let me expand the context to see the surrounding line.   Is there something unusual about the way your created this PR?

Reply:
Oh!   That feature looks like it doesn't work in "ignore whitespace" mode.

Reply:
Umm now its working.   Ignore me.